products/memgraph/upcoming/tutorials/got-deaths.md

```diff
@@ -0,0 +1,232 @@
+    * a "Character" node has a "name" attribute corresponding to the character's name (e.g. "Castle Black")
+  * edges, of type "KilledBy", "Killed","LoyalTo", "VictimIn","KillerIn", "InEpisode", "InSeason", or "InLocation"
+    * edges "KilledBy" and "Killed" connect two Character nodes and they have 2 attributes, 
+      "method" which says how was the character killed (e.g. "Knife") and "count" attributes which says
+      how many of these characters was killed (e.g. if "Jon Snow" killed 10 "Lannister soldiers" then on 
+      this path "count" would be "10")
+    * edge "LoyalTo" connects "Character" node with "Allegiance" node
+    * edges "VictimIn" and "KillerIn" connects "Character" node with "Death" node
+    * edge "InLocation" connects node "Death" with "Location" node
+    * edge "InEpisode" connects node "Death" node "Episode"
+    * edge "InSeason" connects nodes "Death" and "Episode" with "Season" node
+
+### Importing the Snapshot
+
+We have prepared a database snapshot for this example, so the user can easily
+import it when starting Memgraph using the `--data-directory` option.
+
+```plaintext
+sudo -u memgraph \
+  /usr/lib/memgraph/memgraph --data-directory /usr/share/memgraph/examples/Got \
+    --storage-snapshot-interval-sec=0 --storage-wal-enabled=false \
+    --storage-snapshot-on-exit=false --storage-properties-on-edges=true
+```
+
+When using Memgraph installed from DEB or RPM package, the currently running
+Memgraph server may need to be stopped before importing the example. The user
+can do so using the following command:
+
+```plaintext
+systemctl stop memgraph
+```
+
+When using Docker, the example can be imported with the following command:
+
+```plaintext
+docker run -p 7687:7687 \
+  -v mg_lib:/var/lib/memgraph -v mg_log:/var/log/memgraph -v mg_etc:/etc/memgraph \
+  memgraph --data-directory /usr/share/memgraph/examples/Got \
+  --storage-snapshot-interval-sec=0 --storage-wal-enabled=false \
+  --storage-snapshot-on-exit=false --storage-properties-on-edges=true
+```
+
+The user should note that any modifications of the database state will persist
+only during this run of Memgraph.
+
+### Example Queries using OpenCypher
+
+In the queries below, we are, as usual, using [OpenCypher](https://www.opencypher.org)
+to query Memgraph via the console.
+
+Here are some queries you might find interesting:
+
+**MINI-GAME** - If you have watched the TV Show, try to guess each result before executing the query!
+1) Let's start with a couple of simple queries. List the locations with the most deaths occurred.
+Can you guess which one is it?
+
+```opencypher
+MATCH (l:Location)<-[:InLocation]-(d:Death) 
+return l.name AS LocationName, COUNT(d) AS DeathCount
+ORDER BY DeathCount DESC
+```
+
+2) Now that we have the location with the most deaths, let's list the episodes with the most deaths
+as well.
+
+```opencypher
+MATCH (d:Death)-[:InEpisode]->(e:Episode)

```

Comment:
To make model simpler, it is good to skip word of the next label in edge type whenever possible. This `()-[]-()` should be read as a sentence where this one, and previous one sounds like `Death In Episode In Episode`. 

I would suggest `:HAPPENED_IN` and that one is used from Death to both Location and Episode. Because that way I can query `(d:Death { value: "something" })-[:HAPPENED_IN]-(n)` and in `n` I can get both Location and Episode. If I want only one of it I would add one change `(n:Location)`.